### ISSUE TITLE
Group chat by interest

### ISSUE THREAD

**Issue by Nonononoki** [Alovoa/alovoa#50]
Users can chat with other users over a specific topic, unbound by user preferences or location

**Comment by davidak** [comment]
How is that related to dating? Why not use the [Fediverse](https://www.youtube.com/watch?v=S57uhCQBEk0) for that?

**Comment by natrius** [comment]
Has nothing to do with dating. Do not reinvent the wheel, there is matrix https://en.m.wikipedia.org/wiki/Matrix_(protocol) for that.

**Comment by aha999** [comment]
Most definitely, this feature would be important so users can talk in real time about Alovoa development or other topics... like a small community. I think that the best platform for this would be Matrix, try the [Element](https://element.io/) software, it's pretty good and popular.

I dared to create a Chat on Matrix, if you join (main dev @Nonononoki ?) i will make you an admin and i will demote myself to a regular user. if you don't like this i can delete the community. Or i can help you with moderating.

**Comment by ctlw83** [comment]
You might say that discussions or group chats about a specific topic or keyword has nothing to do with dating, but if we think about dating in the context of pre-internet dating, that is EXACTLY how people found each other.

People found other people through church, movie groups, book clubs, paranormal research teams, debate clubs, walking groups, gaming groups, anime groups, geocaching, etc.

Having open discussion groups with clear boundaries that there is "no dating talk" in the chat or group, but would allow you to find other people with similar interests, hover over their profile, and then like the profile and start a private chat outside of the public discussion would be the equivalent of being in a club or group in real life and then asking a person in the group out after getting to know them a little.

**Comment by jeffrey734** [comment]
Yeah exactly Ctlw83, this can be interesting for some users just like the app Boo does. There's a public place where people can share messages. It shows people's personality when they share public comments and helps them to get in contact with others. I like the idea and it should be done correctly. I think that those apps aren't only dating apps but also can we say "meeting" apps? You can find people for different reasons. Bumble does it for jobs and friends. Facebook for friends too. Also Tinder has different modes to find different kinds of people based on their interests. There's definitely something to explore there.

**Comment by ctlw83** [comment]
@jeffrey734 MeetUp just announced a feature like this where if you are at an event and meet someone you are interested in connecting with you can say you want to connect with them.  If they, independently of you, thought you were interesting and decided they wanted to connect with you, they can do the same.  If both people, independently of each other, tell MeetUp that they want to connect, it allows them to connect and talk through the app.

Granted, this is more meant on a shared recent event than just a conversation or interest, but similar concept.
